tests/store/test_obssurface.py

```diff
@@ -904,6 +913,23 @@ def test_gcwerks_fp_not_a_tuple_raises():
             obs.read_file(filepath=filepath, source_format="gc", site="cgo", network="agage")
 
 
-def test_get_store_path():
-    bucket = get_bucket()
-    print(bucket)
+def test_object_loads_if_invalid_objectstore_path_in_json(tmpdir):
```

Comment:
Could you add docstring to say what this is testing for? e.g. modifying `_bucket` in the data type metadata (in ObsSurface/uuid/<some uuid>._data) doesn't affect what buckets are used for searching and standardising? Maybe add a link to Joe's issue too? (I can understand the test from your comments, but out of context it might be a bit confusing.)

Reply:
Good idea.